pyproject.toml

```diff
@@ -0,0 +1,36 @@
+[build-system]
+requires = ["setuptools>=61.0"]
+build-backend = "setuptools.build_meta"
+
+[tool.setuptools]
+packages = ["swiftsimio"]
+
+[project]
+name = "swiftsimio"
+version="6.1.1"
+authors = [
+    { name="Josh Borrow", email="joshua.borrow@durham.ac.uk" },
+]
+description="SWIFTsim (swift.dur.ac.uk) i/o routines for python."
+readme = "README.md"
+requires-python = ">3.8.0"
+classifiers = [
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "License :: OSI Approved :: GNU Lesser General Public License v3 or later (LGPLv3+)",
+    "Operating System :: OS Independent",
+]
+dependencies = [
+    "numpy",
+    "h5py",
+    "unyt>=2.9.0",
```

Comment:
Do we want to show `unyt>=3.0.0` now?

Reply:
Actually I don't see unyt >=3.0.0 on PyPI even on python 3.11...

Reply:
Ah right unyt3.0.0 has not yet been released. My bad!